collector/mongod/metrics.go

```diff
@@ -397,7 +397,7 @@ type CursorStats struct {
 // Export exports the cursor stats.
 func (cursorStats *CursorStats) Export(ch chan<- prometheus.Metric) {
 	metricsCursorTimedOutTotal.Set(cursorStats.TimedOut)
-	metricsCursorOpen.WithLabelValues("noTimeout").Set(cursorStats.Open.NoTimeout)
+	metricsCursorOpen.WithLabelValues("timed_out").Set(cursorStats.Open.NoTimeout)
```

Comment:
Are we sure noTimeout means timed_out,  sounds like that means its not timed out to me